src/quo2/components/settings/settings_list/style.cljs

```diff
@@ -0,0 +1,49 @@
+(ns quo2.components.settings.settings-list.style
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.platform :as platform]))
+
+(defn title
+  []
+  {:color (colors/theme-colors
+           colors/neutral-100
+           colors/white)})
+
+(def icon
+  {:margin-right 12})
+
+(def item-container
+  {:justify-content    :space-between
+   :height             48
+   :flex-direction     :row
+   :align-items        :center
+   :padding-horizontal 12
+   :padding-vertical   13})
+
+(def switch-button
+  (if platform/ios?
+    {:transform [{:scaleY 0.588} {:scaleX 0.645}]}
```

Comment:
Why do we need this scaling?

Reply:
This was used to modify the React Native Switch component size, but since we are using quo2.toggle now we can delete it.